web_hide_user_menu_item/README.rst

```diff
@@ -0,0 +1,74 @@
+=======================
+Web Hide User Menu Item
+=======================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/15.0/web_hide_user_menu_item
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-15-0/web-15-0-web_hide_user_menu_item
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/15.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module hides documentation, support, shortcuts, account menu items on the top-right corner of the webclient.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+After using this module, you will see only preferences and logout menu items like below image.
+    .. image:: https://raw.githubusercontent.com/Aungkokolin1997/web/1111-web_hide_user_menu_item/web_hide_user_menu_item/static/img/screenshot.png
```

Comment:
@AungKoKoLin1997 Please do not add this directly in this file.  Instead, prepare USAGE.rst under `/readme` for this.

> After using this module, you will see only preferences and logout menu items like below image.

This is not necessarily accurate since some modules add items here.  Can we change it to "Upon installing the module, user menu item presentation will be stripped down to look like the below image"?

Also place the image under `/static/img` instead of your own account.

Reply:
Thanks for your suggestion. I fixed it.